build-all.sh

```diff
@@ -3,9 +3,9 @@
 set -eu
 
 NODE_STABLE="16"
-NODE_VERSIONS=${NODE_VERSIONS:="18.10.0 16.17.1 14.20.1 12.22.12"}
+NODE_VERSIONS=${NODE_VERSIONS:="18.12.1 16.18.1 14.21.1"}
```

Comment:
They're still using Node 12 in [firmware](https://github.com/connectedcars/firmware/blob/master/Dockerfile)

Reply:
Node is not updating 12.x anymore so don't know if it makes sense to keep building it.

Reply:
Makes sense not to build it as the old version will stay in the container registry.